homeassistant/components/iotawatt/sensor.py

```diff
@@ -45,63 +45,63 @@ class IotaWattSensorEntityDescription(SensorEntityDescription):
 
 ENTITY_DESCRIPTION_KEY_MAP: dict[str, IotaWattSensorEntityDescription] = {
     "Amps": IotaWattSensorEntityDescription(
-        "Amps",
+        key="Amps",
```

Comment:
This is because of the updated type hint, there's no issue instantiating with positional arguments in runtime. https://github.com/home-assistant/core/pull/105211#discussion_r1419085307